pandas/core/groupby/groupby.py

```diff
@@ -1079,7 +1079,8 @@ def get_group(self, name, obj=None) -> DataFrame | Series:
             raise KeyError(name)
 
         if obj is None:
-            return self._selected_obj.iloc[inds]
+            indexer = inds if self.axis == 0 else (slice(None), inds)
```

Comment:
Just to confirm that we won't need this once `self.axis` is deprecated?

Reply:
Removed, but yes